src/test/groovy/com/ullink/MSBuildTaskBuilder.groovy

```diff
@@ -33,12 +33,12 @@ class MSBuildTaskBuilder {
     }
 
     public MSBuildTaskBuilder withArtifact(String artifactPath) {
-        artifacts.add(new File(artifactPath))
+        artifacts.add(new File(artifactPath.replace('\\', File.separator)))
```

Comment:
I don't this this is such a good idea, is it? You might intentionally have "\" in the path, like when escaping space (at least on linux)